ci.properties

```diff
@@ -329,6 +330,7 @@ ci.test.suite.description[object-as]=Test object tests with different applicatio
 ci.test.suite.description[object-db]=Test object tests with different databases.
 ci.test.suite.description[object-db-partition]=Test object functional tests with Database Partitioning.
 ci.test.suite.description[object-lxc]=Test Object tests with Liferay Experience Cloud configurations.
+ci.test.suite.description[object-functional]=Test Object tests with only functional tests.
```

Comment:
Please follow the existing patterns for the descriptions.

Reply:
I'll follow the same pattern from `commerce-functional` then.